bin/resources/GameIndex.yaml

```diff
@@ -32434,17 +32434,17 @@ SLPM-62160:
   name: "Simple 2000 Series Vol. 4 - The Double Mahjong Puzzle"
   region: "NTSC-J"
 SLPM-62161:
-  name: GENERATION OF CHAOS NEXT [限定版]
-  name-sort: GENERATION OF CHAOS NEXT [げんていばん]
+  name: GENERATION OF CHAOS NEXT (限定版) 
```

Comment:
Trailing whitespace causing the gamedb check to fail

Reply:
Sorry